File: ella/week4/M_1436.py
# 브루트 포스 알고리즘(완전 탐색 알고리즘)

n = int(input())
num = 666 # 첫번재 영화 숫자
cnt = 0 # 666이 들어가는 숫자 count

while True:
    if "666" in str(num): # 연속 666이 있어야 함, 없을 경우 count X
        cnt += 1
    if cnt == n: # n번째 영화 숫자까지 count해서 반복문 탈출
        break
    num += 1

print(num)

File: leo/week4/M-1436.js
/**
 *  제목 : 영화감독 숌
 *  입력 : 첫 줄에 N이 주어진다 (1 ≤ N ≤ 10,000)
 *  출력 : N번째로 작은 종말의 수를 출력한다.
 */

// Run by Node.js
const readline = require("readline");

(async () => {
  const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout,
  });

  const input = await new Promise((resolve) => rl.question("", resolve));
  const N = parseInt(input.trim(), 10);

  let count = 0;
  let number = 666;

  while (true) {
    if (String(number).includes("666")) {
      count += 1;
    }
    if (count === N) {
      console.log(number);
      break;
    }
    number += 1;
  }

  rl.close();
  process.exit();
})();
